server/api/index.js

```diff
@@ -6,13 +6,42 @@
  * it can be accessed via /:resource
  */
 import {Router as router} from 'express';
+import {BigQuery} from '@google-cloud/bigquery';
+import mixedApi from './mixed-content';
 
 // TODO(ernestognw): Setup routes for every model in db
 const api = router();
+const bigqueryClient = new BigQuery();
 
 // Health check
 api.get('/', (req, res) => {
   res.status(200).send('API working');
 });
 
+// Health check (BigQuery)
```

Comment:
Health check is a common practice for REST APIs, and usually is created only once, since it is only used to monitor services. You can [check this out](https://www.ibm.com/garage/method/practices/manage/health-check-apis/) to get more info.

That being said, I saw you're not using this endpoint. If you're planning to use it in the future, just change the Healthcheck comment. If not I think we can safely remove the whole endpoint.

Let me know what you think about this.

Reply:
Yes, I will remove it. Originally it was intended to test the bigquery connection, but that can be tested doing a simple request to some real endpoint.